server/auth/sso/sso.go

```diff
@@ -227,11 +227,9 @@ func (s *sso) HandleRedirect(w http.ResponseWriter, r *http.Request) {
 func (s *sso) HandleCallback(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	state := r.URL.Query().Get("state")
-	cookie, err := r.Cookie(state)
-	http.SetCookie(w, &http.Cookie{Name: state, MaxAge: 0})
-	if err != nil {
-		w.WriteHeader(400)
-		return
+	cookie, cookieErr := r.Cookie(state)
+	if cookieErr != nil {
+		http.SetCookie(w, &http.Cookie{Name: state, MaxAge: 0})
```

Comment:
It looks like this will change the default path, won't it? I assume the typical case is that `cookieErr == nil`. Before we were setting `http.SetCookie()` in that case, and now we are not. Right?